src/pages/_app.tsx

```diff
@@ -37,6 +38,15 @@ const MyApp: AppType<{ session: Session | null }> = ({
   const router = useRouter();
   const isHomePage = router.pathname === pages.home;
 
+  const { width } = useViewport();
+  const [isMobile, setIsMobile] = useState(false);
+
+  useEffect(() => {
+    if (width) {
+      setIsMobile(width <= breakpoints.md);
```

Comment:
Why do we need to set this in two different places? Wouldn't the most top container be enough?

Reply:
The Homepage is isolated from the entire game. The top container rules do not apply to the homepage because it needs it to consume the entire screen with no padding and cannot be scrollable. Think of the homepage as a huge poster.

Reply:
Ahh clear...